### DISCUSSION TITLE
Why access token does not includes client_id or azp?

### DISCUSSION THREAD

**Discussion by duizabojul** [zitadel/oidc#646] [Q&A] [answered]
id_token got these claims but not access_token? Is there a reason for this? [RFC tells client_id is required in access token](https://datatracker.ietf.org/doc/html/rfc9068#name-data-structure).

**Comment by muhlemmer** [comment, marked as answer]
Hi, good question. You are right.

It seems we forget to assign the value here:

https://github.com/zitadel/oidc/blob/52e8b651d36e638d462a00162af7fab16cb74734/pkg/oidc/token.go#L107-L123

I will convert this issue into a bug report and submit a fix. Thanks for raising!
